cosmo_tester/test_suites/test_blueprints/hello_world_bash_test.py

```diff
@@ -92,7 +92,14 @@ def _run(self, image_name, user, is_existing_deployment=False):
 
         floating_ip_id, neutron, nova, sg_id, server_id =\
             self._verify_deployment_installed()
-        self.assert_monitoring_data_exists()
+        self.assert_deployment_monitoring_data_exists(self.test_id)
```

Comment:
then here you dont have to pass anything to the assert_bla_bla...because this test does not use a deployment id of its own